src/platforms/launchpad-icdi/platform.c

```diff
@@ -48,6 +49,11 @@ uint32_t platform_time_ms(void)
 	return time_ms;
 }
 
+int platform_hwversion(void)
```

Comment:
Is there a particular reason for removing this from the platform header? This will impair the optimiser's ability to optimise this function out to just a value in all call sites